etc/bin/run-functional-tests

```diff
@@ -183,6 +183,9 @@ export TMPDIR
 # test environment
 export CYLC_TEST_SKIP=${CYLC_TEST_SKIP:-}
 CYLC_TEST_TIME_INIT="$(date -u +'%Y%m%dT%H%M%SZ')"
+CYLC_TEST_UID="$(< /dev/urandom tr -dc _A-Z-a-z-0-9 | head -c4)"
```

Comment:
Give each test battery run a four character identifier to distinguish it from other runs initiated at the same time.

This is important if running the same tests on multiple platforms simultaneously.